EIPS/eip-4750.md

```diff
@@ -110,11 +121,12 @@ Under `PC_post_instruction` we mean the PC position after the entire immediate a
 In addition to container format validation rules above, we extend code section validation rules (as defined in [EIP-3670](./eip-3670.md)).
 
 1. Code validation rules of EIP-3670 are applied to every code section.
-2. List of allowed *terminating instructions* in EIP-3670 is extended to include `RETF`. (*Note that `CALLF`, like other instructions with immediates, cannot be truncated.*)
-3. Code section is invalid in case an immediate argument of any `CALLF` is greater than or equal to the total number of code sections.
-4. `RJUMP` and `RJUMPI` immediate argument value (jump destination relative offset) validation:
+2. List of allowed *terminating instructions* in EIP-3670 is extended to include `RETF` and `TAILCALLF`. (*Note that `CALLF` and `TAILCALLF`, like other instructions with immediates, cannot be truncated.*)
+3. Code section is invalid in case an immediate argument of any `CALLF` or `TAILCALLF` is greater than or equal to the total number of code sections.
+4. Code section is invalid in case an immediate argument of any `TAILCALLF` is such that `type[callee_section_index] != type[caller_section_index]`, i.e. it is allowed to tail-call only the function with the same output type. 
```

Comment:
Should that be
```suggestion
4. Code section is invalid in case an immediate argument of any `TAILCALLF` is such that `type[callee_section_index].outputs != type[caller_section_index].outputs`, i.e. it is allowed to tail-call only the function with the same output type. 
```
?

Reply:
Yes, exactly, thank you